nemo/core/config/optimizers.py

```diff
@@ -0,0 +1,89 @@
+# =============================================================================
+# Copyright (c) 2020 NVIDIA. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+
+from dataclasses import dataclass
+from typing import Tuple
+
+__all__ = ['AdamParams', 'AdamConfig', 'NovogradParams', 'NovogradConfig']
```

Comment:
let's merge this as-is for now. But I have a feeling @titu1994 will need to re-do this in his next PR

Reply:
Yep, this alongside schedulers will get refactored